bin/GameIndex.dbf

```diff
@@ -21477,6 +21477,11 @@ Compat = 5
 Serial = SLPM-62718
 Name   = Sega Ages 2500 Series Vol.27 - Panzer Dragoon
 Region = NTSC-J
+[patches = 8D984276]
+	// By PSI
+	comment=Stop screen shaking (game switches to Progressive)
```

Comment:
I prefer
```
comment=Patch by PSI
// Stop screen shaking (game switches to Progressive).
```

Reply:
I'd retain what patch actually is; not author

Reply:
All the other patches (most) are done in the format I suggested.

Reply:
I dislike that idea; Consistency is important, but I think it'd better to show what patch actually does.
Does it show both?